### PR TITLE
Update act-rules-format.bs for secondary req and acc support

### PR DESCRIPTION
**Pull request by kengdoj**
Add Secondary requirements and accessibility support conclusion from https://github.com/act-rules/act-rules.github.io/discussions/2095

### CODE REVIEW

File: act-rules-format/act-rules-format.bs [hunk 1]
```diff
@@ -226,6 +226,7 @@ A secondary accessibility requirement is a requirement that is correlated with t
 
 When the rule is not designed to test the accessibility requirement, or failed outcomes of the rule still require further testing for the accessibility requirement, the rule <em class="rfc2119">may</em> map the accessibility requirement as Secondary. When an ACT rule maps to a Secondary requirement, it <em class="rfc2119">must</em> include an explanation of why that requirement is Secondary in the Background section of the rule.
 
+When the rule is designed to test an accessibility requirement, limitations or differences in [accessibility support](#accessibility-support) shall not be a reason for that requirement to be a secondary accessibility requirement.
```
Comment (tbostic32) [inline act-rules-format/act-rules-format.bs:229]: Should we replace "shall" with either "should" or "must"?

Comment (daniel-montalvo) [reply act-rules-format/act-rules-format.bs:229]: +1 to "should"

File: act-rules-format/act-rules-format.bs [hunk 2]
```diff
@@ -226,6 +226,7 @@ A secondary accessibility requirement is a requirement that is correlated with t
...
```
Comment (daniel-montalvo) [inline act-rules-format/act-rules-format.bs:230]: Not t sure about the word "limitations" here. Does not "differences" express by itself what we mean here?

**Review by daniel-montalvo — CHANGES_REQUESTED** [review]

File: act-rules-format/act-rules-format.bs (hunk 1 above)
Comment (kengdoj) [inline act-rules-format/act-rules-format.bs:229]:
```suggestion
When the rule is designed to test an accessibility requirement, limitations or differences in [accessibility support](#accessibility-support) should not be a reason for that requirement to be a secondary accessibility requirement. 
```

**Review by daniel-montalvo — APPROVED** [review]

**Review by tbostic32 — APPROVED** [review]

**Pull request merged**